### ISSUE TITLE
Not sending automatically.

### ISSUE THREAD

**Issue by Vanderdoody** [halfmatthalfcat/wotd#2]
The bot doesn’t send automatically. I have to use /wotd to get it to go so I know it has permissions to send messages. I’ve done /setup and it says it was successful. I let it go all weekend to see if maybe the time was off, but it’s not sending unless I /wotd.
![IMG_0987](https://github.com/halfmatthalfcat/wotd/assets/78441431/3563c034-4948-4d57-b3f9-045f51926fb1)
![IMG_0988](https://github.com/halfmatthalfcat/wotd/assets/78441431/d9f32212-fbaa-44e5-8b86-8896ec13259a)

**Comment by Vanderdoody** [comment]
So just checked again today, even after taking that screenshot where it says it was set up successfully it still didn't send.

**Comment by halfmatthalfcat** [comment]
@Vanderdoody hmm let me look into it. Thanks for letting me know.

**Comment by halfmatthalfcat** [comment]
@Vanderdoody can you send me your guild channel name? If you don't wanna do it in here, you can DM me in Discord @halfmatthalfcat

Everything looks to be working on my end and it looks like we've been sending out wotd successfully to all other guilds

**Comment by Vanderdoody** [comment]
The server name is Fort Castle.

**Comment by Vanderdoody** [comment]
Strange. I did /setup again a day or two ago and it’s still not working. It send just fine when I use /wotd so I know it has permissions. Guess it wasn’t meant to be.

**Comment by dominik-dev** [comment]
@Vanderdoody. Can you see "Word of the Day" bot **online** in the text channel you setup WotD? I ran into this problem and created a new text channel where I can see him online and it worked fine then.

**Comment by Vanderdoody** [comment]
Yup. It was online and in the text channel. It was sending just fine when I’d use /wotd. But it wouldn’t send automatically. I tried everything and decided it was just a subpar bot and gave up on it.
